### ISSUE TITLE
Implement loss function

### ISSUE THREAD

**Issue by LuXoN1999** [LuXoN1999/Segmentation-of-abdominal-organs-from-medical-images#13]
Define loss function which will be used in training process.

# TODO
- implement loss function which will be a combination of:  
  - Binary Cross Entropy (BCE)
  - Dice coefficient

# DOCUMENTATION
https://towardsdatascience.com/understanding-binary-cross-entropy-log-loss-a-visual-explanation-a3ac6025181a
https://en.wikipedia.org/wiki/Dice-Sørensen_coefficient
